extensions/wikia/Email/templates/userNameLayout.mustache

```diff
@@ -0,0 +1,20 @@
+{{> greeting}}
+<table>
+	<tr>
+		<td class="border">
+			<div class="avatar-wrapper">
+				<img class="avatar" src="{{editorAvatarURL}}">
+			</div>
+		</td>
+		<td class="widest border">
+			<div class="message">
+				<p>
+					{{userNameMessage}}
+				</p>
+			</div>
+		</td>
+	</tr>
+	{{# hasContentFooterMessages}}
```

Comment:
What I mean is that I don't think we need to have this `{{# hasContentFooterMessages}}` check in this template since we're always going to display the `{contentFooterMessages}}`.

In the avatarLayout template this check makes sense since it's shared by a number of emails, some of which that show contentFooterMessages and some that don't. If this template is only used by emails which show contentFooterMessages, we don't have to have that check.